hoomd/md/ForceComposite.h

```diff
@@ -73,7 +73,50 @@ class PYBIND11_EXPORT ForceComposite : public MolecularForceCompute
          */
         virtual void validateRigidBodies(bool create=false);
 
-    protected:
+        
+        /// Construct from a Python dictionary
+        void setBody(std::string typ, pybind11::object v)
+            {   
+            pybind11::list types = v["types"];
+            pybind11::list positions = v["positions"];
+            int N = (unsigned int)len(positions);
+
+            // extract the positions from the python list
+            std::vector<vec3<Scalar>> pos_vector;
+            for (unsigned int i = 0; i < N; i++)
+                {   
+                pybind11::list pos_i = positions[i];
+                if (len(pos_i) != 3)
+                    throw std::runtime_error("Each position must have 3 coordinates");
+                vec3<OverlapReal> pos = vec3<Scalar>(pybind11::cast<Scalar>(pos_i[0]),
+                                                          pybind11::cast<OverlapReal>(pos_i[1]),
+                                                          pybind11::cast<OverlapReal>(pos_i[2]));
+                pos_vector.push_back(pos);
+                }   
+
+            }   
+
+        /// Convert parameters to a python dictionary
+        pybind11::dict getBody(std::string typ)
```

Comment:
Why is there a `getBody` if there is no way to query this info in python?

Reply:
Check Josh's comment from Jan 26. I'm not familiar with C++ so my progress on this is very slow. I think the plan is to add methods in python that query them.